src/app/ubs/ubs-admin/components/ubs-admin-several-orders-pop-up/ubs-admin-several-orders-pop-up.component.ts

```diff
@@ -102,6 +103,8 @@ export class UbsAdminSeveralOrdersPopUpComponent implements OnInit {
 
   showTimePickerClick(): void {
     this.showTimePicker = true;
+    this.fromInput = this.ordersForm.get('exportDetailsDto').get(FormFieldsName.TimeDeliveryFrom).value;
```

Comment:
Repeated code in setExportTime and showTimePickerClick can be optimized to prevent duplication.
 The chain of invocation of get() can lead to a potential null or undefined value, which can cause runtime errors

```
private getFormControl(groupName: string, fieldName: string): AbstractControl | null {
  const group = this.ordersForm.get(groupName);
  return group ? group.get(fieldName) : null;
}
```